service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/utils/DateFormatUtil.java

```diff
@@ -1,76 +1,83 @@
 package uk.nhs.adaptors.gp2gp.ehr.utils;
 
-import static ca.uhn.fhir.model.api.TemporalPrecisionEnum.DAY;
-import static ca.uhn.fhir.model.api.TemporalPrecisionEnum.MINUTE;
-import static ca.uhn.fhir.model.api.TemporalPrecisionEnum.MONTH;
-import static ca.uhn.fhir.model.api.TemporalPrecisionEnum.YEAR;
-
 import java.time.Instant;
 import java.time.ZoneId;
 import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeFormatterBuilder;
-import java.util.Calendar;
-import java.util.Date;
 
 import org.hl7.fhir.dstu3.model.BaseDateTimeType;
 
-import com.google.common.collect.ImmutableMap;
-
-import ca.uhn.fhir.model.api.TemporalPrecisionEnum;
 import uk.nhs.adaptors.gp2gp.ehr.exception.EhrMapperException;
 
 public class DateFormatUtil {
+    public static final int NINE = 9;
```

Comment:
What these constants represent? Maybe we could give them more specific names?

Reply:
i represents the number nine